pkgs/applications/virtualization/kvmtool/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, fetchurl, lib }:
+
+stdenv.mkDerivation {
+  pname = "kvmtool";
+  version = "2021-12-14";
```

Comment:
iirc we've sort of standardized on prefixing versions with "unstable-" for software that does not do "proper" releases

Reply:
Ok